packages/design-system-docs/src/pages/components/ChoiceList/checkbox-radio.example.html

```diff
@@ -11,7 +11,7 @@
       checked
     />
     <label for="checkbox-1">
-      Checkbox 1
+      <span>Checkbox 1</span>
```

Comment:
Added span tags around the examples in this doc to be consistent with the React versions